mri/operators/utils/__init__.py

```diff
@@ -7,10 +7,10 @@
 # for details.
 ##########################################################################
 
-"""
-This module contains shortcuts to linear classes.
+""" This module defines the common operators.
 """
 
-# Package import
-from mri.reconstruct.linear import WaveletN
-from mri.reconstruct.linear import WaveletUD2
+# Fourier utils
+from ..fourier.utils import convert_mask_to_locations, \
+    convert_locations_to_mask, normalize_frequency_locations, \
+    get_stacks_fourier
```

Comment:
Why not also link to linear utils?

Reply:
Missed it, cause as of now we dont use linear utils. But will add them.

Reply:
I don't think you need to comment these imports, the names are pretty self-explanatory.

Reply:
Well now that they are in, I think I will keep them there. Anyway it creates distinction in code lines for future.